composer.json

```diff
@@ -17,8 +17,10 @@
     },
     "require-dev": {
         "phpunit/phpunit": "~8.5.0",
+        "laminas/laminas-diactoros": "^2.2.2",
```

Comment:
This is unneeded, laminas is a dependency of cakephp itself.

Reply:
Travis CI failed after removed this dependency. Some suggestion?

Reply:
Bump up the `cakephp/cakephp` version constraint to `^4.0.2`